docs/_dg/UseCases.md

```diff
@@ -122,9 +122,9 @@ MSS:
 
 #### Use Case 6: Add Tag to Item
 
-Preconditions: Tag has been created in FoodRem.
+Preconditions: [[ tag:Tag ]] has been created in FoodRem.
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```